AirCasting/AirCastingApp.swift

```diff
@@ -8,22 +8,25 @@
 import SwiftUI
 import Firebase
 import Combine
+
 @main
 struct AirCastingApp: App {
     @Environment(\.scenePhase) var scenePhase
     private let authorization: UserAuthenticationSession
     private let syncScheduler: SynchronizationScheduler
     private let microphoneManager: MicrophoneManager
+    private let appState: AppState
     private var sessionSynchronizer: SessionSynchronizer
     private let persistenceController = PersistenceController.shared
     private let appBecameActive = PassthroughSubject<Void, Never>()
     @ObservedObject private var offlineMessageViewModel: OfflineMessageViewModel
     private var cancellables: [AnyCancellable] = []
-
+//    let appState = AppState()
```

Comment:
Why did you leave this comment? :)

Reply:
I cannot see it, maybe from earlier commit? 🙁